src/__tests__/core/block.test.ts

```diff
@@ -15,6 +15,56 @@ test('block: select single block by number', async () => {
   expect(result).toEqual(expected);
 });
 
+test('block: select single transaction from block by index', async () => {
+  const query =  `
+    {
+        block(number: 5771878) {
+          difficulty
+          transactionAt(index: 0) {
+            inputData
+            status
```

Comment:
For lines 24 and 25, check a different field for value rather than a null field. Hash is a good one to check.

Reply:
Same goes for the tests below